addons/mail/models/res_config_settings.py

```diff
@@ -43,3 +45,31 @@ def _compute_fail_counter(self):
             ('date', '>=', previous_date),
             ('state', '=', 'exception'),
         ])
+
+    def open_mail_notification_paynow(self):
+        layout_id = self.env.ref('mail.mail_notification_paynow').id
+        return {
+            'type': 'ir.actions.act_window',
+            'name': 'Mail Layout',
+            'view_mode': 'form',
+            'res_id': layout_id,
+            'res_model': 'ir.ui.view',
+        }
+
+    def open_message_notification_email(self):
+        layout_id = self.env.ref('mail.message_notification_email').id
+        return {
+            'type': 'ir.actions.act_window',
+            'name': 'Mail Layout',
+            'view_mode': 'form',
+            'res_id': layout_id,
+            'res_model': 'ir.ui.view',
+        }
+
+    def open_mail_templates(self):
```

Comment:
Can't you use an existing action (xml record) instead of this generic one that has no helper, ... ? We could use action_email_template_tree_all (that has no helper while looking at it, maybe this is the time to write one :D ).